Resources/Inconsistencies.tex

```diff
@@ -0,0 +1,88 @@
+\documentclass[11pt,fleqn]{article}
+
+\newcommand{\indentpar}{\phantom{=}}
+\newcommand{\bli}{\begin{itemize}}
+\newcommand{\eli}{\end{itemize}}
+
+\begin{document}
+
+    %% Title Page %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+    \begin{center}
+
+        {\large \textbf{VDispl}}\\[2mm]
+        {\large \textbf{Soil Settlement Analysis Software}}\\[2mm]
+        {\huge \textbf{Inconsistencies in Settlements Textbook and FORTRAN Code}}\\[6mm]
+        {\large \textbf{Dr.~Spencer Smith, Emil Soleymani}}\\[2mm]
+        {\large \textbf{McMaster University}}\\[6mm]
+        {\large Revised: May 9, 2022}
+      
+      \end{center}
+      
+      \medskip
+
+      % Description
+      \indentpar During review of the legacy FORTRAN code for the VDispl software, 
+      we referenced the Settlements Analysis book linked in the README 
+      file in order to correlate each calculation with its corresponding 
```

Comment:
Whenever you write "in order to" you can probably replace it with "to".  This is an example of a low information content phrase.  You can find examples of things to watch for like this in my [writing checklist](https://gitlab.cas.mcmaster.ca/smiths/cas741/-/blob/master/BlankProjectTemplate/docs/Writing-Checklist.pdf).

Reply:
Sentence changed